src/Console/ResetCommand.php

```diff
@@ -5,12 +5,10 @@
 use Doctrine\DBAL\Connection;
 use Exception;
 use Illuminate\Console\Command;
-use Illuminate\Console\ConfirmableTrait;
 use LaravelDoctrine\Migrations\Configuration\ConfigurationProvider;
 
 class ResetCommand extends Command
 {
-    use ConfirmableTrait;
```

Comment:
Instead of removing, I'd prefer to see

```
if (!$this->confirmToProceed()) {
    return;
}
```

added. 

Running this in production should give a warning.